### ISSUE TITLE
Player improvement - songs loading & volume

### ISSUE THREAD

**Issue by the-me-0** [the-me-0/groove#12]
Two things : 
- the songs are mp3 files, that the player has to load entirely before starting to play them. This provokes a long waiting time before the songs actually start playing on clients with a slow/medium bandwidth. Adding to the issue, there is nothing in the UI/UX to tell the user that a song is currently loading.
- the volume seems to reset each time the song switch to another one. It does not reset the true volume setting, only the visual "progress bar" for the volume. UI/UX incoherence with setting. Adding to that, the volume edit is too limited ; only ten volume values are possible (0%, 10%, 20%, ..., 100%) and it would be a better user experience to have a smoother slider for this volume.

**Comment by the-me-0** [comment]
The volume slider incoherence has been fixed in [this commit](https://github.com/the-me-0/groove/commit/886059aba789ec6660139b374926fa052b9c0fab), and will be pushed into production in the next version release.
About the songs loading time, I could come with a temporary update for the UX with a loading icon, as I'm currently looking for solutions adressing this issue.

About the volume slider, I believe it's a limitation of howlerjs, the library under the hood of our player. I'm currently thinking about switching to another player to remove this kind of limitations.

**Comment by the-me-0** [comment]
I'm currently making significant progress on the player update. Switching to using the standard html audio tag allows to automatically play the song while it is still downloading, while also allowing a better control of the volume.
I am updating the player components to work with this new audio tag as we speak, with the progress bar and volume bar already working.

**Comment by the-me-0** [comment]
A little update here !
I'm still working on the new player, and I happy to confirm that it is well on the way ! There is still some work to do, but play/pause, skip, previous, loop, smoother volume & smoother progress bar are already done. Further testing is requiredfor those big features are they are vital to the application, but I should push a preview really soon.

As I'm working on an upgraded player, I figured I would try to implement a "big picture" mode for desktop too ; I'll keep this issue updated.

**Comment by the-me-0** [comment]
Standard player fully done ; big picture ongoing

**Comment by the-me-0** [comment]
> As I'm working on an upgraded player, I figured I would try to implement a "big picture" mode for desktop too ; I'll keep this issue updated.

I came back on the project to finish this, and got rid of the desktop big picture thing on the way. Player is similar to before, but everything is just a lot smoother. Load time has been reduced, volume is smooth.

All that has been brought to develop with the last merge (just above), and will be part of the next release !
